packages/bcer-api/app/src/location/entities/location.entity.ts

```diff
@@ -97,6 +97,12 @@ export class LocationEntity {
   @JoinTable()
   manufactures: ManufacturingEntity[];
 
+  @Column('boolean', {
+    nullable: false,
+    default: false,
+  })
+  closed: boolean;
```

Comment:
Thoughts on making this a `status` instead? (`CLOSED | ACTIVE | (DELETED)`)

Reply:
Probably covered by VAPE-383, but we should probably add a `closedAt` timestamp to record when the location was closed